wire-ios-sync-engine/Source/Use cases/E2EIdentityCertificateUpdateStatusUseCase.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public enum E2EIdentityCertificateUpdateStatus {
+    case noAction, reminder, block
+}
+
+// sourcery: AutoMockable
+public protocol E2EIdentityCertificateUpdateStatusProtocol {
```

Comment:
Maybe rename the protocol?
```suggestion
public protocol E2EIdentityCertificateUpdateStatusUseCaseProtocol {
```

Also put it into a separate file please.

Reply:
Not sure if we are going to gain anything by putting this dependency in a separate file. It will just take more time to compile.